neptune/new/run.py

```diff
@@ -56,6 +56,51 @@
 
 
 class Run(AbstractContextManager):
+    """A Run in Neptune is a representation of all metadata that you log to Neptune.
+
+    Beginning when you start a tracked run with `neptune.init()` and ending when the script finishes
+    or when you explicitly stop the experiment with `.stop()`.
+
+    You can log many ML metadata types, including:  TODO
+        * metrics
+        * losses
+        * model weights
+        * images
+        * interactive charts
+        * predictions
+        * and much more
+
+    Have a look at the complete list of metadata types you can log.
```

Comment:
This and the following two lines either let's remove or add proper linking. I would propose removing them all together.

Reply:
ok